### ISSUE TITLE
Adding hits from other databases in addition to nt and RefProt. 

### ISSUE THREAD

**Issue by st303118** [blobtoolkit/blobtools2#12]
I am currently cleaning my transcriptomic data using blobtools. As described in manual I used blast nt and Uniprot database hits.
Can I add nr blast hits to create a blobdir. If yes, do I need to run diamond blastx or just blastx will be okay? Thank you.

**Comment by rjchallis** [comment]
Yes, you can use any BLAST database so nr will be fine. You'll need the taxonomic information to be present so straight blastx would be easier, but diamond blastx will run faster once you have the db formatted.

once you have the results file, just pass it to `blobtools add` with the `--hits` flag.

**Comment by eddydowle** [comment]
Is there a way to manually add the taxonomy in for a blastn file? I have a weird species that doesnt match much on the standard databases but there is a few draft genomes and the blasts against those look useful, but they are unannotated genomes (so I cant do a blastx). Blobtools will let me add the blast file in but it wont show up in the viewer, I assume because the taxonomy isnt incorporated, can I add the taxonomy in for them (there is only three of these draft genomes)?
